tools/scripts/insert-dropdown-url.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash -ex
+
+# check for dirty changes
+if [[ -n "$(git status --porcelain)" ]]; then
+    echo "untracked or dirty files are not allowed, cleanup before running insert-dropdown-url.sh"
+    exit 1
+fi
+
+# insert-version-url.py inserts a new entry into the versions.json--adding a dropdown link
+python3 docs/insert-version-url.py $(grep -oE '^[0-9.]*' VERSION)
```

Comment:
```suggestion
python3 docs/insert-version-url.py "$(grep -oE '^[0-9.]*' VERSION)"
```
Doesn't make a difference here, but prefer not to have to think about whether it does.